report/export.py

```diff
@@ -1 +1,24 @@
+import reports
 # Export functions
+
+def export(source_file_name,output_file_name):
```

Comment:
nem mukodik, mert ha .__doc__  -ot hasznalom, akkor az csak ugy mukodik, ha a fuggveny-t parameterek nelkul irom ele, de nekem az nem jo, mert van ahol kulonbozoek a parameterek, szal mindenkepp a listaba is parameterekkel egyutt tarolhatom a fuggvenyeket.szted?